src/components/Login/Login.jsx

```diff
@@ -22,12 +24,15 @@ const Login = () => {
 		if (areTrue) {
 			const response = await loginService(userInfo);
 			if (response.status === 201) {
-				localStorage.setItem(
-					'token',
-					response.data.result.replace('Bearer ', '')
+				const token = response.data.result.replace('Bearer ', '');
+				dispatch(
```

Comment:
Local storage logic should be reverted, as after page refresh - you will be like logged out

Reply:
will, do in the next task